static/src/js/tree/index.jsx

```diff
@@ -1,26 +1,37 @@
 "use strict";
 
 import ReactDOM from "react-dom";
-// Although React is not used in this module, if this line is removed, there is
-// a runtime error in codelistbuilder.jsx.
 import React from "react";
 
 import Hierarchy from "../hierarchy";
-import Tree from "./Tree";
+import TreeTables from "../tree-tables";
 import { readValueFromPage } from "../utils";
 
+
 const hierarchy = new Hierarchy(
   readValueFromPage("parent-map"),
   readValueFromPage("child-map")
 );
 
+const treeTables = readValueFromPage("tree-tables");
+const codeToStatus = readValueFromPage("code-to-status");
+const codeToTerm = readValueFromPage("code-to-term");
+
+const ancestorCodes = treeTables
+  .map(([_, ancestorCodes]) => ancestorCodes) // eslint-disable-line no-unused-vars
```

Comment:
Does JS not have a way to ignore variables when looping?? (arg!)

Reply:
Turns out there is!  Fixed in 2405a22f.